index.html

```diff
@@ -149,42 +141,24 @@ <h2 class="Tonic"> Multi-Post <br>Stories</h2>
     <img class="Counter" src="image/Counter7.png" alt="">
     <li><a href="#" class="A">2015</a> </li>
   </ul>
-
-
-
-  <p class="Main-Text5">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
-
- <div class="Languages5">
+   <p class="Main-Text5">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+   <div class="Languages5">
       <button type="submit" value="Submit" class="lan51">html</button>
       <button type="submit" value="Submit" class="lan52">css</button>
       <button type="submit" value="Submit" class="lan53">javascript</button>
   </div>
 
-
   <div class="See-Project5">
     <button type="submit" value="Submit" class="btn3">See Project</button>
   </div>
```

Comment:
- [ ] Your `See Project` and `Get resume` button doesn't have the button state styling implemented, you are meant to implement the `hover and pressed` state for your button according to the Figma design.  I'll suggest you implement this for all the right buttons in your project.

- - FIGMA DESIGN BUTTON STATES:
![Screenshot from 2023-06-02 12-18-10](https://github.com/Tshepo45/Mobile-Finish/assets/107632780/307ca63b-dd31-4238-b96a-c24f41397da3)